content/en/cosign/overview.md

```diff
@@ -63,23 +63,71 @@ $ cosign verify-blob <file> --bundle cosign.bundle --certificate-identity=name@e
 
 To verify, Cosign queries the transparency log (Rekor) to compare the public key bound to the certificate, and checks the timestamp on the signature against the artifact’s entry in the transparency log. The signature is valid if its timestamp falls within the small window of time that the key pair and certificate issued by the certificate authority were valid.
 
-### Working with containers
+## Working with containers
 
-Signing and verifying a container is similar to working with blobs. The Cosign command to sign a container image is:
+> NOTE: You will need access to a container registry for cosign to work with. [ttl.sh](https://ttl.sh/) offers free, short-lived (i.e., hours), anonymous container image hosting if you just want to try these commands out.
```

Comment:
I don't think we need this content in the overview, lines 68-77. I would suggest removing it entirely, I don't think it adds anything. If someone does think there's value, then we could put it in signing_with_containers.md.

Reply:
rewrote to make this an example.